### ISSUE TITLE
kein Sound, Soundchip kann nicht aktiviert werden

### ISSUE THREAD

**Issue by Bibergoahead** [feren-OS/Bug-Reporting-Center#321]
[code]
System:    Kernel: 5.15.0-107-generic x86_64 bits: 64 compiler: N/A Desktop: KDE Plasma 5.25.5 
           tk: Qt 5.15.6 wm: kwin_x11 dm: LightDM Distro: Feren OS 20.04 2023.04 
Machine:   Type: Laptop System: Apple product: MacBookAir4,2 v: 1.0 serial: <filter> Chassis: 
           type: 10 v: Mac-742912EFDBEE19B3 serial: <filter> 
           Mobo: Apple model: Mac-742912EFDBEE19B3 v: MacBookAir4,2 serial: <filter> UEFI: Apple 
           v: 135.0.0.0.0 date: 06/14/2019 
Battery:   ID-1: BAT0 charge: 15.6 Wh condition: 16.0/50.2 Wh (32%) volts: 8.2/7.5 
           model: SMP bq20z451 serial: N/A status: Full 
CPU:       Topology: Dual Core model: Intel Core i5-2557M bits: 64 type: MT MCP arch: Sandy Bridge 
           rev: 7 L2 cache: 3072 KiB 
           flags: avx lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx bogomips: 13599 
           Speed: 2346 MHz min/max: 800/2700 MHz Core speeds (MHz): 1: 2400 2: 2612 3: 2400 
           4: 2301 
Graphics:  Device-1: Intel 2nd Generation Core Processor Family Integrated Graphics vendor: Apple 
           driver: i915 v: kernel bus ID: 00:02.0 chip ID: 8086:0116 
           Display: x11 server: X.Org 1.20.13 driver: modesetting unloaded: fbdev,vesa 
           compositor: kwin_x11 resolution: 1440x900~60Hz 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 3000 (SNB GT2) v: 3.3 Mesa 21.2.6 
           compat-v: 3.0 direct render: Yes 
Audio:     Device-1: Intel 6 Series/C200 Series Family High Definition Audio driver: snd_hda_intel 
           v: kernel bus ID: 00:1b.0 chip ID: 8086:1c20 
           Sound Server: ALSA v: k5.15.0-107-generic 
Network:   Device-1: Broadcom and subsidiaries BCM43224 802.11a/b/g/n 
           vendor: Apple AirPort Extreme driver: bcma-pci-bridge v: N/A port: efa0 bus ID: 02:00.0 
           chip ID: 14e4:4353 
           IF-ID-1: wlp2s0b1 state: up mac: <filter> 
Drives:    Local Storage: total: 447.13 GiB used: 9.20 GiB (2.1%) 
           ID-1: /dev/sda vendor: OWC model: Aura Pro 6G SSD size: 447.13 GiB speed: 6.0 Gb/s 
           serial: <filter> 
Partition: ID-1: / size: 430.64 GiB used: 9.19 GiB (2.1%) fs: ext4 dev: /dev/sda2 
           ID-2: swap-1 size: 8.24 GiB used: 0 KiB (0.0%) fs: swap dev: /dev/sda3 
USB:       Hub: 1-0:1 info: Full speed (or root) Hub ports: 6 rev: 2.0 chip ID: 1d6b:0002 
           Hub: 1-1:2 info: Microchip (formerly SMSC) 2.0 Hub ports: 3 rev: 2.0 chip ID: 0424:2513 
           Hub: 1-1.1:4 info: Broadcom BCM2046B1 USB 2.0 Hub (part of BCM2046 Bluetooth) ports: 3 
           rev: 2.0 chip ID: 0a5c:4500 
           Device-1: 1-1.1.3:8 info: Apple Built-in Bluetooth 2.0+EDR HCI type: <vendor specific> 
           driver: btusb rev: 2.0 chip ID: 05ac:821f 
           Device-2: 1-1.2:5 info: Apple FaceTime Camera (Built-in) type: Keyboard,HID,Mouse 
           driver: apple,bcm5974,usbhid rev: 2.0 chip ID: 05ac:024c 
           Device-3: 1-2:3 info: Apple FaceTime Camera type: Video driver: uvcvideo rev: 2.0 
           chip ID: 05ac:850a 
           Hub: 2-0:1 info: Full speed (or root) Hub ports: 8 rev: 2.0 chip ID: 1d6b:0002 
           Hub: 2-1:2 info: Microchip (formerly SMSC) 2.0 Hub ports: 2 rev: 2.0 chip ID: 0424:2513 
           Device-4: 2-1.1:3 info: Apple Internal Memory Card Reader type: Mass Storage 
           driver: usb-storage rev: 2.0 chip ID: 05ac:8404 
           Device-5: 2-1.2:4 info: Holtek Optical Mouse type: Mouse driver: hid-generic,usbhid 
           rev: 1.1 chip ID: 04d9:0499 
           Hub: 3-0:1 info: Full speed (or root) Hub ports: 2 rev: 1.1 chip ID: 1d6b:0001 
           Hub: 4-0:1 info: Full speed (or root) Hub ports: 2 rev: 1.1 chip ID: 1d6b:0001 
Sensors:   System Temperatures: cpu: 61.0 C mobo: N/A 
           Fan Speeds (RPM): N/A 
Repos:     Active apt repos in: /etc/apt/sources.list 
           1: deb http: //archive.ubuntu.com/ubuntu focal main restricted universe multiverse
           2: deb http: //archive.ubuntu.com/ubuntu focal-updates main restricted universe multiverse
           3: deb http: //archive.ubuntu.com/ubuntu focal-backports main restricted universe multiverse
           4: deb http: //security.ubuntu.com/ubuntu/ focal-security main restricted universe multiverse
           5: deb http: //archive.canonical.com/ubuntu/ focal partner
           Active apt repos in: /etc/apt/sources.list.d/additional-repositories.list 
           1: deb https: //dl.winehq.org/wine-builds/ubuntu/ focal main
           Active apt repos in: /etc/apt/sources.list.d/feren-os-neon.list 
           1: deb https: //gitlab.com/feren-os/feren-repositories-neon-focal/raw/master stable neon-focal
           Active apt repos in: /etc/apt/sources.list.d/feren-os.list 
           1: deb https: //gitlab.com/feren-os/feren-repositories-focal/raw/master stable main software
           Active apt repos in: /etc/apt/sources.list.d/google-chrome.list 
           1: deb [arch=amd64] http: //dl.google.com/linux/chrome/deb/ stable main
           No active apt repos in: /etc/apt/sources.list.d/graphics-drivers-ppa-focal.list 
           Active apt repos in: /etc/apt/sources.list.d/vivaldi.list 
           1: deb [arch=amd64] https: //repo.vivaldi.com/stable/deb/ stable main
Info:      Processes: 220 Uptime: 23m Memory: 3.73 GiB used: 1.37 GiB (36.8%) Init: systemd v: 245 
           runlevel: 5 target: graphical.target Compilers: gcc: 9.4.0 alt: 9 
           Client: Unknown python3.8 client inxi: 3.0.38 
[/code]

**Comment by dominichayesferen** [comment]
Sounds like a hardware-specific incompatibility - also can't reproduce given this, so... this will have to have these tags for now unless someone with your hardware can help debug this one:

**Comment by dominichayesferen** [comment]
Does this still persist in Feren OS 2025.03?

**Comment by Bibergoahead** [comment]
no, thank you - is working all right 👍Von meinem iPhone gesendetAm 28.04.2025 um 22:06 schrieb Dominic Hayes - Feren OS Dev ***@***.***>:﻿
dominichayesferen left a comment (feren-OS/Bug-Reporting-Center#321)
Does this still persist in Feren OS 2025.03?
